### PR TITLE
Buckets sorting [storage]

### LINKED ISSUE #2174: [Storage] Add sort in Buckets table
**Is your feature request related to a problem? Please describe.**
Should be good to have the sorting in that table, because we have it in all other tables, so in this way we maintain a standard.

**Describe the solution you'd like**
Add the ability to sort by:
- Name
- File System
- Size

**Additional context**

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #2174

I have not placed any sorting by default.
Sorting only triggered on column head clicks.

### CODE REVIEW

**Review by asnaith — APPROVED** [review]
👍 This is a nice improvement, working great in chrome and firefox

File: packages/storage-ui/src/Components/Pages/BucketsPage.tsx [hunk 1]
```diff
@@ -121,16 +121,21 @@ const useStyles = makeStyles(({ breakpoints, animation, constants, typography }:
   })
 )
 
+type SortColumn = "name" | "file_system" | "size" | undefined
```
Comment (Tbaut) [inline packages/storage-ui/src/Components/Pages/BucketsPage.tsx]: undefined is already an option in the sortColumn state
```suggestion
type SortColumn = "name" | "file_system" | "size"
```

**Comment by juans-chainsafe** [comment]
Working great! good job

**Review by Tbaut — APPROVED** [review]
Looks good, just a nit

**Pull request merged**